src/cube_fit/cube_fit.py

```diff
@@ -2,6 +2,7 @@
 
 
 def find_cube_transform(face_points_1: np.ndarray, face_points_2: np.ndarray, face_points_3: np.ndarray):
+    
```

Comment:
Since this is the main library function to be used, it should definitely have a docstring. Something like:

```suggestion
    """
    Find the transformation matrix from a collection of surface points on three orthogonal faces of a cube
    
    :param face_points_1: Nx3 numpy array of points
    :param face_points_2: Nx3 numpy array of points
    :param face_points_3: Nx3 numpy array of points
    :return: The 4x4 transformation matrix of the orthogonal edge basis vectors from the corner point of the cube
    """
```